app/models/TulsaBlotter/arrest.rb

```diff
@@ -0,0 +1,5 @@
+class TulsaBlotter::Arrest < ApplicationRecord
+  belongs_to :inmate, optional: true
```

Comment:
Are you sure this doesn't need to have the foreign_key listed since these are namespaces?
Also, I'd say this shouldn't be optional since you need to have an inmate for there to be an arrest